demos/dashy/style/index.css

```diff
@@ -248,4 +248,8 @@
   color: #9E9E9E;
   font-size: 11px;
   font-weight: 400;
+}
+
+.layout_Grid > .laneBackground {
+  display: none;
```

Comment:
Should this be in the layout_Grid css file?

Reply:
Ultimately this just allows us to ignore the fact that the `.laneBackground` has some `box-sizing: content-box` sizing issues because of its 1 pixel border. A real fix for this would be one of these two:
- move that style to layout_Grid's css file like you suggested, but change it to give laneBackground `border: 0` (I'll test this solution)
- adjust the layout_Cell placement+sizing logic to account for the widths of the cell borders and laneBackground borders (both in and out of designMode)